### ISSUE TITLE
Typescript pipelines

### ISSUE THREAD

**Issue by canadaduane** [lmnr-ai/lmnr#251]
I see pipeline "Code" nodes can have code written in Python. Will it be possible to write "Code" nodes in typescript?

**Comment by skull8888888** [comment]
@canadaduane Yes! We're actively working on it. Is there a specific reason to use TS over Python or are you just more comfortable with TS?

**Comment by canadaduane** [comment]
I was exploring/contrasting Lmnr and Dify. Currently our team of ~10 engineers is typescript-focused due to our building a user-friendly nextjs app for students and teachers.

**Comment by skull8888888** [comment]
interesting, I didn't know you could write code in Dify. Are you just building a prototype or you are considering using either Dify or Laminar pipeline builder for production use case?

**Comment by canadaduane** [comment]
It's a production use case. We are switching everything over from a "code + prompt registry" style of workflow to a combined visual workflow.

A comment I made to another product team that was building a prompt registry and eval framework we had been considering:

> The main job-to-be-done is organizing representations of "what is happening" inside the black box so that both engineers and prompt specialists can co-create, understand, critique, and collaborate. I think Dify handles that at a higher level that made sense to both teams--as long as we generate a type-level contract around the boundary between Dify workflows and the rest of our app, the compiler can also help us stay on track.
